Update user guide (#69)
Jelly-RDF/jelly-rdf.github.io: docs/user-guide.md

But, what if our stream contains a sequence of RDF graphs or datasets? For example, if we want to transfer the measurements of a temperature sensor, we would have:

Graph 1:

```turtle
ex:sensor ex:measuredProperty ex:temperature ;
    ex:unit ex:Kelvin ;
    ex:value "280.4"^^xsd:decimal .
```

Graph 2:

```turtle
ex:sensor ex:measuredProperty ex:temperature ;
    ex:unit ex:Kelvin ;
    ex:value "280.6"^^xsd:decimal .
```

...and so on. In traditional RDF systems, you had two options. You could modify the data, wrap it in containers, or apply some other processing to make it fit into one graph. Or you could simply put each graph/dataset in a separate file. Jelly offers a simpler way, where multiple graphs or datasets can live within one file. We use *frames* as boundary markers:

```turtle
# Frame 1
ex:sensor ex:measuredProperty ex:temperature ;
    ex:unit ex:Kelvin ;
    ex:value "280.4"^^xsd:decimal .

# Frame 2
ex:sensor ex:measuredProperty ex:temperature ;
    ex:unit ex:Kelvin ;
    ex:value "280.6"^^xsd:decimal .
```

> We call this a *grouped RDF stream*. A Jelly parser can unpack this and process the frames one by one. The best part is that the compression (explained [above](#how-does-it-work--encoding)) is applied **across the entire stream**. So, if an IRI appears in frame 1, and then again in frame 2, we will only have to write it only once. This is very effective for regular data, like IoT measurements, nanopublications, encyclopedic entries, or maps (geography).

Author: No, "regular" as in "with repeating patterns"

> We call this a *grouped RDF stream*. A Jelly parser can unpack this and process the frames one by one. The best part is that the compression (explained [above](#how-does-it-work--encoding)) is applied **across the entire stream**. So, if an IRI appears in frame 1, and then again in frame 2, we will only have to write it only once. This is very effective for regular data, like IoT measurements, nanopublications, encyclopedic entries, or maps (geography).

Reviewer: AHA
then maybe
```suggestion
We call this a *grouped RDF stream*. A Jelly parser can unpack this and process the frames one by one. The best part is that the compression (explained [above](#how-does-it-work--encoding)) is applied **across the entire stream**. So, if an IRI appears in frame 1, and then again in frame 2, we will only have to write it only once. This is very effective for data with repeating patterns, like IoT measurements, nanopublications, encyclopedic entries, or maps (geography).
```
just to remove the ambiguity that the word "regular" introduces?

> We call this a *grouped RDF stream*. A Jelly parser can unpack this and process the frames one by one. The best part is that the compression (explained [above](#how-does-it-work--encoding)) is applied **across the entire stream**. So, if an IRI appears in frame 1, and then again in frame 2, we will only have to write it only once. This is very effective for data with repeating patterns, like IoT measurements, nanopublications, encyclopedic entries, or maps (geography).

Reviewer: ```suggestion
We call this a *grouped RDF stream*. A Jelly parser can unpack this and process the frames one by one. The best part is that the compression (explained [above](#how-does-it-work--encoding)) is applied **across the entire stream**. So, if an IRI appears in frame 1, and then again in frame 2, we will only have to write it once. This is very effective for data with repeating patterns, like IoT measurements, nanopublications, encyclopedic entries, or maps (geography).
```

Author: let's keep the "only".... it's fine, it's emphasis

- **`QUADS`**: Data is encoded using quad statements. Each quad has a graph name, which can also be the default graph.
- **`GRAPHS`**: Data is encoded using named graphs, where the graph name can also be the default graph. Each named graph can contain multiple triples.

!!! warning

>     The `GRAPHS` type is a left-over from a previous version of the format and right now it's usefulness is limited – you can do the exact same thing with `QUADS`, which is simpler. In new projects, we recommend using `QUADS` instead.

Reviewer: ```suggestion
    The `GRAPHS` type is a left-over from a previous version of the format and right now its usefulness is limited – you can do the exact same thing with `QUADS`, which is simpler. In new projects, we recommend using `QUADS` instead.
```
